### DISCUSSION TITLE
安装程序过程报错，失败，大佬看看报错代码怎么回事？

### DISCUSSION THREAD

**Discussion by 3384966915** [lsky-org/lsky-pro#382] [问答]
Dropped all tables successfully.
Migration table created successfully.
Migrating: 2014_10_10_000000_create_groups_table
Installation error!

SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'json not null comment '组配置', `created_at` timestamp null, `updated_at` tim' at line 1 (SQL: create table `groups` (`id` bigint unsigned not null auto_increment primary key, `name` varchar(64) not null comment '角色组名称', `is_default` tinyint(1) not null default '0' comment '是否默认', `is_guest` tinyint(1) not null default '0' comment '是否为游客组', `configs` json not null comment '组配置', `created_at` timestamp null, `updated_at` timestamp null) default character set utf8mb4 collate 'utf8mb4_unicode_ci' engine = InnoDB)
![Screenshot_2022-04-09-09-42-12-841_com ume browser hs](https://user-images.githubusercontent.com/103295718/162552054-b1bdf77c-e39b-4471-8c47-6985f6d889ca.jpg)

**Comment by akatsukiro** [comment]
您就不能看一下前面的Discussion？

**Reply by akatsukiro** [reply]
https://github.com/lsky-org/lsky-pro/discussions/357

**Comment by Qiuxiaoye007** [comment]
请使用MySQL 5.7/MariaDB 10.2 及以上版本。

**Reply by anzhiyu-c** [reply]
我感觉产生这个问题，是因为宝塔默认安装的5.7.37版本的mysql也会报这个错，所以很多人报这个

**Reply by akatsukiro** [reply]
> 我感觉产生这个问题，是因为宝塔默认安装的5.7.37版本的mysql也会报这个错，所以很多人报这个

我刚刚使用宝塔安装了MySQL 5.7.37 进行安装，没有报错。

**Reply by Qiuxiaoye007** [reply]
在宝塔的软件商店搜索的MySQL显示是5.7版本，而点进去进行切换我第一次看见的是显示的5.6版本
也就是说我在第一次安装的时候宝塔默认使用的是5.6版本，所以报了以上错误
之后点入进行版本切换为5.7版本这个问题也就解决了
